### ISSUE TITLE
Slight Intermittent Image Difference

### ISSUE THREAD

**Issue by karlhorky** [cypress-visual-regression/cypress-visual-regression#147]
Hi there 👋 Thanks for maintaining this plugin, it's very useful!

I wanted to reach out because we have started to have intermittent problems with our visual regression tests failing.

It will always be on the same image, and this image will have just slight differences (no visual difference 🤔)

The diff will look like this:

<img width="596" alt="Screenshot 2022-12-14 at 21 39 53" src="https://user-images.githubusercontent.com/1935696/207710170-7ab518d8-192e-4806-9d9a-9593c56465ea.png">

Then we will run the tests again, and because this bug is intermittent, it will pass (with 50% probability)

Interestingly enough, when it does fail, the `pixelmatch` threshold number is exactly the same:

```
  1) Visual regression tests
       should show a matching homepage:
     Error: The "home" image is different. Threshold limit exceeded! 
Expected: 0 
Actual: 0.006637992698643086
      at compareSnapshotsPlugin (/home/runner/work/upleveled.io/upleveled.io/node_modules/cypress-visual-regression/dist/plugin.js:81:13)
```

Versions of dependencies we're using:

- Cypress version `12.0.2`
- Electron version `106`
- `cypress-visual-regression` version `1.7.0`
- `cypress-io/github-action` version `5.0.0`

Any clue as to what this could be?

**Comment by karlhorky** [comment]
After some investigation, it seems like we have found the issue: the [`will-change` CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/will-change) on a parent.

> The will-change [CSS](https://developer.mozilla.org/en-US/docs/Web/CSS) property hints to browsers how an element is expected to change. Browsers may set up optimizations before an element is actually changed. These kinds of optimizations can increase the responsiveness of a page by doing potentially expensive work before they are actually required.

![Screenshot 2022-12-14 at 21 47 49](https://user-images.githubusercontent.com/1935696/207711234-afc4b41b-71e5-40f1-a95d-cc2b45d4c9fd.png)

We introduced this in order to improve the performance of the carousel that it was in, but this change has caused the element to be sometimes rendered differently (causing the image difference of `0.006637992698643086`).

Our guess is that one of the latest Electron / Chromium updates has caused `will-change` to only sometimes be applied (maybe also for performance reasons), which means that the rendering of the image will be different on different test runs.

We removed this CSS property (and then we needed to update the snapshot) and it has eliminated the problem.
